packages/theme/src/theme/_colors.scss

```diff
@@ -80,6 +80,11 @@ $chestnut-rose: #e96065;
 /// @type Color
 $lightning-yellow: #f3c446;
 
+/// Secondary color.
+///
+/// @type Color
+$jaffa: #ea8330;
```

Comment:
https://www.color-hex.com/color/ea8330

Reply:
maybe just maybe some info coming from the UX team stating how this new colour should be used.